src/javascript/EditPanel/EditPanelContent/FormBuilder/Section/FieldSet/Field/SelectorTypes/Picker/Search/Search.jsx

```diff
@@ -5,19 +5,24 @@ import {Input} from '@jahia/design-system-kit';
 import {Search} from '@material-ui/icons';
 import {FolderDisplayer} from './FolderDisplayer';
 
-export const SearchInput = ({selectedPath, placeholder, ...props}) => {
+export const SearchInput = ({selectedPath, placeholder, language, ...props}) => {
     return (
         <Input
             type="text"
             data-sel-role="data-sel-picker-search"
             placeholder={placeholder}
-            variant={{icon: <Search/>, interactive: <FolderDisplayer selectedPath={selectedPath}/>}}
+            variant={{icon: <Search/>, interactive: <FolderDisplayer selectedPath={selectedPath} language={language}/>}}
             {...props}
         />
     );
 };
 
+SearchInput.defaultProps = {
+    language: 'en'
+};
+
 SearchInput.propTypes = {
     selectedPath: PropTypes.string.isRequired,
-    placeholder: PropTypes.string.isRequired
+    placeholder: PropTypes.string.isRequired,
+    language: PropTypes.string
```

Comment:
I think we shoud force to send a language to this component
```suggestion
    language: PropTypes.string.isRequired
```

Reply:
I don't think so, and at least we have default value of it.

What did you think?